template/app/src/client/app/AccountPage.tsx

```
@@ -70,6 +51,66 @@ export default function AccountPage({ user }: { user: User }) {
   );
 }
 
+function UserCurrentSubscriptionStatus(user: User) {
+  const prettyPrintSubscriptionPlan = (userSubscriptionPlan: User['subscriptionPlan']) => {
+    const capitalizeFirstLetter = (str: string) => str.charAt(0).toUpperCase() + str.slice(1);
+    if (!userSubscriptionPlan) console.error('User subscription plan is missing');
+    if (userSubscriptionPlan === SubscriptionPlanId.Hobby) {
+      return capitalizeFirstLetter(SubscriptionPlanId.Hobby);
+    }
+    if (userSubscriptionPlan === SubscriptionPlanId.Pro) {
+      return capitalizeFirstLetter(SubscriptionPlanId.Hobby);
+    }
```

What if we didn't match any of these two plans? We should throw an error! Maybe we were given some silly value?

Also, we have the switch/record situation here -> if new subscription plan is added, we won't detect here that we are missing implementation for it. So we shoudl use Record approach here.